numaprom/udsink/train_rollout.py

```diff
@@ -139,19 +145,24 @@ def train_rollout(datums: List[Datum]) -> Responses:
             responses.append(Response.as_success(_datum.id))
             continue
 
-        if len(train_df) < win_size:
+        if len(train_df) < model_cfg.conf["seq_len"]:
```

Comment:
I think we can replace this with checking against MIN_TRAIN_SIZE. And maybe replace this with the check we have in `clean_data()`. We can remove the check there. What do you think?